### DISCUSSION TITLE
Imported tags from Calibre (bibtex.file) are a single tag in Zotero

### DISCUSSION THREAD

**Discussion by gieselbusch** [retorquere/zotero-better-bibtex#2743] [General]
Hi,

I'm trying to import my collection of PDFs from Calibre using bibtex export and import.
In Calibre a added tags to many PDFs, separated by a `,` and mostly followed by a single white space.
After the import. All tags are a single tag in Zotero.
Is there any way to change this?

Thank you!

**Comment by retorquere** [comment]
Do you have a sample?

**Reply by gieselbusch** [reply]
I'm sorry, I should have thought of that!
Attached you'll find several files:
- [Test.bib](https://pastebin.com/raw/d9u3aRqg) -> Export from Calibre
- [Test-export.bib](https://pastebin.com/raw/yL8QT9gV) -> Export from Zetoero (Better BibTeX)
- "better BibTex.json" -> my current Better BibTeX Settings
Thank you!


[better BibTeX.json](https://github.com/retorquere/zotero-better-bibtex/files/13872453/better.BibTeX.json)

Screenshot_01 -> How it looks like after the import in Zotero
![Screenshot_01](https://github.com/retorquere/zotero-better-bibtex/assets/66715494/93fe66cf-e9f6-4908-bc7a-1d09fd9077f9)

Screenshot_02 -> All tags are one single tag.
![Screenshot_02](https://github.com/retorquere/zotero-better-bibtex/assets/66715494/c149e01e-fdd9-458a-89ff-14fbef952412)

Screenshot_03 -> How I edited the tags and expect them to be
![Screenshot_03](https://github.com/retorquere/zotero-better-bibtex/assets/66715494/6312ce4c-cde6-4be3-bc11-16e8ffe68cd8)

**Comment by retorquere** [comment]
I have a fix available -- I will make a new release in a few days, if you're in a hurry, open an issue at https://github.com/retorquere/zotero-better-bibtex/issues and I can get you a version ahead of time with the fix.

**Comment by gieselbusch** [comment]
Thank you for your fast answer! That's really a great help!
I can wait for the official version.

Have a nice weekend
